tests/conftest.py

```diff
@@ -1,6 +1,10 @@
 import pytest
 from scrapy.settings import Settings
 
+from scrapy_poet.page_input_providers import ResponseDataProvider
+
+ResponseDataProvider.register()
```

Comment:
Don't remember this well. Didn't we have a method `install()` somewhere where all the providers where installed?

I'm a little bit concerned about having a static list for the providers. I have the feeling that is better to have it somewhere else (e.g. the middleware), and then have in the modules some kind of installation function that can be used to fill it.

Reply:
Ohh, I see this is the test.

Reply:
No static list of providers anymore. Marking as fixed.